magento/adobe-stock-integration
# Add user picture to the profile popup

### Description (*)
![DeepinScreenshot_select-area_20191003155838](https://user-images.githubusercontent.com/2481206/66128552-bc935b00-e5f6-11e9-98af-3b21fea57796.png)
![DeepinScreenshot_select-area_20191003155847](https://user-images.githubusercontent.com/2481206/66128553-bd2bf180-e5f6-11e9-9f12-1b5e70b714a1.png)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#474: Add user picture to the profile popup
2. ...

### Manual testing scenarios (*)

1. Login to admin panel
2. Open Magento Media Gallery (i.e. go to Cms -> Pages, edit the page and insert image)
3. Click "Search Adobe Stock" button to open images grid
4. Login to Adobe using the "Sign In" link
5. Click on your name to reveal user profile popup

User avatar should be added to the profile information
